core-primitives/pallet-storage/src/lib.rs

```diff
@@ -23,12 +27,28 @@ impl StoragePrefix for EnclaveBridgeStorage {
 
 pub trait EnclaveBridgeStorageKeys {
 	fn shard_status<T: Encode>(shard: T) -> Vec<u8>;
+	fn upgradable_shard_config<T: Encode>(shard: T) -> Vec<u8>;
+	fn pallet_version() -> Vec<u8>;
 }
 
 impl<S: StoragePrefix> EnclaveBridgeStorageKeys for S {
 	fn shard_status<T: Encode>(shard: T) -> Vec<u8> {
 		storage_map_key(Self::prefix(), "ShardStatus", &shard, &StorageHasher::Blake2_128Concat)
 	}
+	fn upgradable_shard_config<T: Encode>(shard: T) -> Vec<u8> {
+		storage_map_key(
+			Self::prefix(),
+			"ShardConfigRegistry",
+			&shard,
+			&StorageHasher::Blake2_128Concat,
+		)
+	}
+
+	fn pallet_version() -> Vec<u8> {
```

Comment:
actually, storage_version would be more precise as it fetches the onchain version, not the oncode version. but PJS still calls it that.